### PR TITLE
added upload and status page

### PR DESCRIPTION
**Pull request by ccanasj**

### CODE REVIEW

File: Frontend/package.json [hunk 1]
```diff
@@ -11,6 +11,8 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "axios": "^1.2.1",
+    "mcmotdparser": "^1.3.1",
+    "minecraft-motd-util": "^1.1.12",
```
Comment (Daves2126) [inline Frontend/package.json]: This library is required?

Comment (Daves2126) [reply Frontend/package.json]: "minecraft-motd-util"

**Review by Daves2126 — CHANGES_REQUESTED** [review]

File: Frontend/package.json (hunk 1 above)
Comment (ccanasj) [reply Frontend/package.json]: No, it's now remove with the current changes

Comment (Daves2126) [reply Frontend/package.json]: Approved @ccanasj

**Review by Daves2126 — APPROVED** [review]

**Pull request merged**